### ISSUE TITLE
Upgrade to actix-web 4

### ISSUE THREAD

**Issue by svenstaro** [orhun/rustypaste#18]
Gotta keep rolling.

**Comment by orhun** [comment]
There is actually a branch that I'm working on to upgrade actix dependencies: https://github.com/orhun/rustypaste/tree/chore/bump_actix_deps

However, there seems to be a couple of issues with the runtime. For example, I'm getting the following error while uploading a file:

```
Requested application data is not configured correctly. View/enable debug logs for more details.
```

So I'm going to hold this a bit until I'm one hundred percent sure that everything is functional after upgrading actix.

Feel free to help testing/debugging 🐻

**Comment by orhun** [comment]
> ```
> Requested application data is not configured correctly. View/enable debug logs for more details.
> ```

This is fixed in 1a07cc672b4b118a4f850a5cfb55c8af7b987d71

Next I will work on the failing tests and then merge it into `main` if everything goes well.
